lib/node_modules/@stdlib/stats/base/dists/logistic/ctor/benchmark/benchmark.js

```diff
@@ -298,16 +351,22 @@ bench( pkg+':skewness', function benchmark( bm ) {
 	var dist;
 	var mu;
 	var s;
+	var len;
 	var y;
 	var i;
```

Comment:
```suggestion
	var len;
	var x;
	var y;
	var i;
```

You should declare all variables at the top that you are using. Make sure you have EditorConfig and linting tools set up locally. That would catch these type of errors.